scripts/generate-dev-environment

```diff
@@ -22,6 +23,7 @@ DIRECT_DEBIT_CONNECTOR_URL=https://localhost:9004
 PRODUCTS_URL=https://localhost:9005
 PUBLIC_AUTH_URL=https://localhost:9006
 LEDGER_URL=https://localhost:9007
+SELFSERVICE_URL=https://selfservice.pymnt.uk
```

Comment:
Was not sure what to put for this value in other environments.
So thought just to link to test env for now.